migrations/2023-09-18-141700_login-token/up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE login_token (
+    id serial PRIMARY KEY,
+    token text NOT NULL UNIQUE,
+    user_id int REFERENCES local_user ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
+    published timestamptz NOT NULL DEFAULT now(),
+    ip text,
+    user_agent text
+);
+
+CREATE INDEX idx_login_token_user ON login_token (user_id);
+
+CREATE INDEX idx_login_token_user_token ON login_token (user_id, token);
```

Comment:
The index idx_login_token_user is unnecessary due to idx_login_token_user_token.

Reply:
I dont know about this, cc @dessalines

Reply:
I spose that's correct, since its the first in the order. You can remove.

Reply:
done